cmd/dlv/cmds/commands.go

```diff
@@ -614,7 +614,7 @@ func traceCmd(cmd *cobra.Command, args []string) {
 }
 
 func isBreakpointExistsErr(err error) bool {
-	return strings.Contains(err.Error(), "Breakpoint exists")
+	return strings.Contains(err.Error(), "breakpoint exists")
```

Comment:
I don't think we should change this, the JSON-RPC API returns strings for errors and older clients may be relying on this formatting.

Reply:
Ok I reverted the changes that are not in delve dap.